src/macros/event_with_options.rs

```diff
@@ -0,0 +1,41 @@
+/// Send a custom event as a title and a body
+/// NOTE: Try to minimise variation in tag values (avoid things like timestamps or ids). See note in lib docs!
+#[macro_export]
+macro_rules! event_with_options {
```

Comment:
WDYT about extending the event! Instead of creating a new one? It could receive the EventOptions as an argument and pass it to `$crate::Datadog::event_with_options` instead of `$crate::Datadog::event`.

Reply:
I was debating the two options, I think it may make sense to expose just one macro